server-plugins/notification-resources/src/utils.ts

```diff
@@ -474,33 +472,50 @@ export async function getUsersInfo (
   ctx: MeasureContext,
   ids: Ref<PersonAccount>[],
   control: TriggerControl
-): Promise<(ReceiverInfo | SenderInfo)[]> {
-  if (ids.length === 0) return []
-  const accounts = await control.modelDb.findAll(contact.class.PersonAccount, { _id: { $in: ids } })
+): Promise<Map<Ref<Account>, ReceiverInfo | SenderInfo>> {
+  if (ids.length === 0) return new Map()
+  const accounts = control.modelDb.findAllSync(contact.class.PersonAccount, { _id: { $in: ids } })
+
   const personIds = accounts.map((it) => it.person)
+  const personIdsMap = new Set(personIds)
   const accountById = toIdMap(accounts)
-  const persons = toIdMap(
-    await ctx.with(
-      'find-persons',
-      {},
-      async () => await control.findAll(contact.class.Person, { _id: { $in: personIds } })
-    )
+  const persons: IdMap<WithLookup<Person>> = new Map()
+  const spaces = new Map<Ref<Person>, PersonSpace>()
+
+  const p = await ctx.with('find-persons', {}, async (ctx) =>
+    (
+      await control.queryFind(
+        ctx,
+        contact.mixin.Employee,
```

Comment:
If we want to find here only `Employes`, so we should find info about notification sender separately, because sender might be not `Employee` (systemAccount, github user)

Now we find info about all users (sender + receivers) in this fn